app/assets/stylesheets/static_pages.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```

Comment:
scss, coffee, helper file -> rails automation generate -> if you don't use it, you can delete it

Reply:
done